docs/content/connectors/redis.md

```diff
@@ -9,7 +9,13 @@ should be deployed in standalone, master-slave, or cluster mode. Currently,
 
 ## Supported Processors and Usages
 
-- Flink: Lookup, Streaming Upsert
+- Flink: Lookup<sup>1</sup>, Streaming Upsert
+
+1. If there is a map-typed feature in Redis and only staticly refered entries in
```

Comment:
It seems unnecessary to document the optimization detail in this doc. This doc should focus on how to use RedisSink, rather than how Redis Sink works.

How about moving this content to https://github.com/alibaba/feathub/blob/master/docs/content/deep-dive/optimizations.md?